api/v1alpha1/ipaddress_types.go

```diff
@@ -45,6 +45,9 @@ type IPAddressSpec struct {
 
 	// Address contains the IP address
 	Address IPAddressStr `json:"address"`
+
+	// DNSServers is the list of dns servers
+	DNSServers []IPAddressStr `json:"dnsServers,omitempty"`
```

Comment:
I'm surprised that it's possible to add new fields to the type here without also updating the generated CRD file.

Reply:
that's my bad, I messed up the commit and that got left out. I'll update. That also shows that we need the generation job here.